src/Shouldly/Shouldly.csproj

```diff
@@ -23,11 +23,11 @@
   <ItemGroup>
     <PackageReference Include="EmptyFiles" Version="4.1.0" PrivateAssets="None" />
     <PackageReference Include="DiffEngine" Version="11.0.0" />
-    <Content Include="build.props" PackagePath="build\Shouldly.props" />
-    <Content Include="buildMultiTargeting.props" PackagePath="buildMultiTargeting\Shouldly.props" />
     <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" Condition="$(Configuration) == 'Release'" />
     <None Include="..\..\assets\logo_128x128.png" Pack="true" PackagePath="assets" />
+  </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' " >
```

Comment:
This is an unrelated change to this PR, but it was a blocker to using Rider, so have bundled up this fix.